docs/source-pytorch/visualize/logging_basic.rst

```diff
@@ -86,7 +89,9 @@ Accumulate a metric
 ===================
 When *self.log* is called inside the *training_step*, it generates a timeseries showing how the metric behaves over time.
 
-.. TODO:: show chart
+.. figure:: https://pl-flash-data.s3.amazonaws.com/assets_lightning/logging_basic/visualize_logging_basic_tensorboard_chart.png
```

Comment:
Maybe have them in the Pl public bucket instead of Flash, which is already archived...

Reply:
What's the name of that bucket? I only know the pl-flash one.

Reply:
https://pl-public-data.s3.amazonaws.com/assets_lightning

Reply:
Moved, thanks!